packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -583,7 +585,9 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     this.context.setCurrentCallId(call.id);
 
     // REMOVE THIS< TEST PURPOSES ONLY
-    call.feature(Features.AcsCaptions).startCaptions();
+    if (_isTeamsCall(call) || (_isACSCall(call) && !call.info.groupId && !call.info.roomId)) {
```

Comment:
Add a one time checker here

Reply:
Will target in next PR

Reply:
Or let's remove this since these lines are not working well in this PR